components/centraldashboard/public/components/namespace-selector.js

```diff
@@ -7,6 +7,12 @@ import '@polymer/paper-item/paper-item.js';
 
 import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
 
+export const ALL_NAMESPACES = 'All namespaces';
+export const ALL_NAMESPACES_ALLOWED_LIST = ['jupyter'];
```

Comment:
@tasos-ale let's have this as an empty list initially, since we don't have an app that supports it at this point in time

Reply:
Ideally I'd like this to info (which web app supports all namespaces) to be transmitted via the common library, so that the dashboard can dynamically know if an app supports this once it loads it.

But we can do with hardcoding the urls for now. If in the future we see a bigger need for this we can look into it